pkg/monitoring/updates/watcher_windows.go

```diff
@@ -1,18 +1,22 @@
 // +build windows
 
-package cagent
+package updates
 
 import (
 	"fmt"
 	"time"
 
 	ole "github.com/go-ole/go-ole"
 	"github.com/go-ole/go-ole/oleutil"
-	log "github.com/sirupsen/logrus"
-
-	"github.com/cloudradar-monitoring/cagent/pkg/common"
 )
 
+type Config struct {
```

Comment:
Why don't use the common updates.Config type both for windows and non-windows environments?
I know that one field is not needed for windows, but it's better than having a copy Config struct.

Reply:
It is not used, forgot to remove